src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/NeuralNetTranches.java

```diff
@@ -0,0 +1,159 @@
+package org.broadinstitute.hellbender.tools.walkers.vqsr;
+
+import htsjdk.tribble.AbstractFeatureReader;
+import htsjdk.tribble.Feature;
+import htsjdk.tribble.FeatureCodec;
+import htsjdk.tribble.TribbleException;
+import htsjdk.tribble.index.Index;
+import htsjdk.tribble.index.IndexFactory;
+import htsjdk.tribble.util.TabixUtils;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.BetaFeature;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.barclay.help.DocumentedFeature;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgram;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.ShortVariantDiscoveryProgramGroup;
+import org.broadinstitute.hellbender.engine.*;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.io.Resource;
+import org.broadinstitute.hellbender.utils.python.PythonScriptExecutor;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Created by sam on 2/5/18.
+ */
+@CommandLineProgramProperties(
+        summary = "Apply tranche filtering",
+        oneLineSummary = "Apply tranche filtering",
+        programGroup = ShortVariantDiscoveryProgramGroup.class
+)
+@DocumentedFeature
+@BetaFeature
+public class NeuralNetTranches extends CommandLineProgram {
+    @Argument(fullName = StandardArgumentDefinitions.INPUT_LONG_NAME,
+            shortName = StandardArgumentDefinitions.INPUT_SHORT_NAME,
+            doc = "Input VCF file")
+    private String inputVcf = null;
+
+    @Argument(fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME,
+            shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
+            doc = "Output VCF file")
+    private String outputVcf = null;
+
+    @Argument(fullName = "snpTruthVcf", shortName = "stv", doc = "Input file of known common SNP sites.")
```

Comment:
Is there a reason these aren't kebab case?

Reply:
fixed